pkgs/applications/science/biology/astral/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, jdk8
+, makeWrapper
+, jre8
+,zip
+}:
+let
+  jdk=jdk8;
+  jre=jre8;
+in
+stdenv.mkDerivation rec {
```

If this is java does it need compilers?

```suggestion
stdenvNoCC.mkDerivation rec {
```

---

check!